modules/components/core/org.wso2.carbon.pc.core/src/main/java/org/wso2/carbon/pc/core/ProcessStore.java

```diff
@@ -1418,6 +1422,70 @@ public String getFlowchart(String flowchartPath) {
         return flowchartString;
     }
 
+    /**
+     *
+     * @param processName
```

Comment:
Params are defined and fixed with commit 762e1814389e8920882601d4b8d7d2363667fe97